external/Invoke-Download.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+  .SYNOPSIS
+  Downloads the Git modules specified in ../.gitmodules
+
+  .DESCRIPTION
+  Parses and downloads the Github repositories specified in the .gitmodules file
+
+  .EXAMPLE
+  PS> .\Get-GitModules.ps1
```

Comment:
I think `external/Get-GitModules.ps` would be a nicer name for this powershell script.

Reply:
will do

Reply:
done